src/dummywallet.cpp

```diff
@@ -50,7 +50,7 @@ void DummyWalletInit::AddWalletOptions(ArgsManager& argsman) const
         "-flushwallet",
         "-privdb",
         "-walletrejectlongchains",
-        "-unsafesqlitesync",
+        "-unsafesqlitesync"
```

Comment:
In b6269b012fe2cad261f94480711c09ebbf47c7fa "wallet: add config to prioritize a solution that doesn't create change in coin selection"

This change is unrelated.

Reply:
Just removed it, thanks!